newIDE/app/src/MainFrame/index.js

```diff
@@ -303,7 +310,20 @@ class MainFrame extends React.Component<Props, State> {
   };
 
   openFromPathOrURL = (url: string, cb: Function) => {
-    const { i18n } = this.props;
+    const { i18n, shouldOpenAutosave } = this.props;
+
+    const projectFilePath = url;
+    const autoSavePath = url + '.autosave';
+    if (shouldOpenAutosave && shouldOpenAutosave(url, autoSavePath, true)) {
+      //eslint-disable-next-line
+      const answer = confirm(
+        i18n._(
+          `Autosave newer than the project file exists. Would you like to load it?`
```

Comment:
Careful, you must use `t` macro, like this:

```
i18n._(t`blabla`)
```

Otherwise "blabla" won't be extracted for translation ;)

Reply:
thank you for fixing this, must have missed it when copying :)